src/main/java/com/org/ita/kata/implementation/MyroslavaVynokur/EigthImpl.java

```
@@ -4,8 +4,11 @@
 
 public class EigthImpl implements Eight {
     @Override
-    public int liters(double time) {
-        return 0;
+    public int liters(double time) throws ArithmeticException{
+        if (time <= 0) {
+            throw new ArithmeticException("time can not be 0");
```

need to throw the error above?

---

removed error